service/src/main/java/com/epam/esm/service/converter/Converter.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.esm.service.converter;
+
+import java.util.List;
+
+public interface Converter<O, D> {
+    /**
+     * Converts Data transfer object to an entity type
+     * @param dto
+     * @return object of a needed type
+     */
+    O dtoObject(D dto);
```

Comment:
https://howtodoinjava.com/java/basics/java-naming-conventions/#:~:text=Method%20naming%20convention,be%20in%20camel%20case%20notation.

Reply:
changed to proper names